container_registry/container_analysis/requirements.txt

```diff
@@ -0,0 +1,2 @@
+google-cloud-pubsub == 0.30.0
+./google_cloud_containeranalysis-0.1.0-py2.py3-none-any.whl
```

Comment:
Included wheel is less common, maybe add a comment:

    # Replace when client library is published.